### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by OGcanviz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,215 @@
-# Project Name
+# Extracting, transforming, and loading data with ACI
 
-(short, 1-3 sentenced, description of the project)
+In this sample, we're trying to do some basic analysis on the [HappyDB](https://github.com/rit-public/HappyDB) dataset, by doing word count and drawing word cloud, to highlight the words appear more frequently in the people's happy moments.
```
Comment (jluk) [inline README.md]: Suggested change with grammatical fixes:
"In this sample, we're analyzing the HappyDB dataset by creating a word cloud of the most frequent words in people's self-reported happiest moments. We'll extract a data source, load it into a database, and transform the data into something we can render graphically."

File: README.md [hunk 2]
```diff
@@ -1,57 +1,215 @@
...
 
-## Features
+**Table of content**
```
Comment (jluk) [inline README.md]: Table of contents (plural)

File: README.md [hunk 3]
```diff
@@ -1,57 +1,215 @@
...
+[References](#references)
 
-- npm install [package name]
-- mvn install
-- ...
+## Foreword
 
-### Quickstart
-(Add steps to get up and running quickly)
+Making ETLing simpler using containers as a plug and play model.
```
Comment (jluk) [inline README.md]: Make ETLing simpler by using Azure Container Instances as a plug and play model.

File: README.md [hunk 4]
```diff
@@ -1,57 +1,215 @@
...
+
+2. Click the container group **MS-ACIAKS-ETLContainerGroups**, notice the state of the first 3 containers is **Terminated**, while the last one is always **Running** to serve the Word Cloud.
+
+   ![](images/deploy-05.png)
+
+3. Copy the **IP address** from the container group blade, and open it in the browser to check the Word Cloud.
+
+   ![](images/deploy-06.png)
+
+### Create Index and Query Data in Azure Shell
```
Comment (jluk) [inline README.md]: Azure Cloud Shell is the product name

File: cmd/extracting/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.9.2 as builder
+WORKDIR /home/src
+COPY . /home/src
+RUN go get github.com/dustin/go-humanize
```
Comment (jluk) [inline cmd/extracting/Dockerfile]: What is this being used for? If this is OSS needed for the demo it needs to be registered.

File: README.md [hunk 6]
```diff
@@ -1,57 +1,215 @@
...
-### Quickstart
-(Add steps to get up and running quickly)
+Making ETLing simpler using containers as a plug and play model.
 
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
+The application with separate common pieces of the ETLing process into separate docker containers. For example, the unzip container in the project will take a link as an input then download and unzip the file at that link. A separate container takes a csv location as an input and puts it into a Postgres database. This allows for plug and play ETLing pipelines for the data.
 
+Using ACI, a user can define container groups with the exact elements they want. For example put the unzip and postgres modules together and to download a zip file from a datasource, unzip it then feed it into a databases all without writing a line of code. This allows you to only pay per second using the ACI instance.
```
Comment (jluk) [inline README.md]: Let's write out ACI as Azure Container Instances.

File: README.md [hunk 7]
```diff
@@ -1,57 +1,215 @@
...
+
+1. Open the resource group you just created.
+
+   ![](images/deploy-04.png)
+
+2. Click the Postgres database, remember the **Server name** and **Server admin login name**.
+
+   ![](images/deploy-07.png)
+
+3. Open the **Cloud Shell** on the top bar.
```
Comment (jluk) [inline README.md]: Specify if you're using Bash or PowerShell. There isn't parity between the two, from your screenshot you're using Bash so let's specifically denote that here.

File: README.md [hunk 8]
```diff
@@ -1,57 +1,215 @@
...
+   ACI_PERS_RESOURCE_GROUP=[RESOURCE_GROUP_NAME]
+   ACI_PERS_STORAGE_ACCOUNT_NAME=mystorageaccount$RANDOM
+   ACI_PERS_LOCATION=eastus
+   ACI_PERS_SHARE_NAME=acishare
+   az group create --location eastus --name $ACI_PERS_RESOURCE_GROUP
+   ```
+
+4. Execute the commands below to create the storage account.
+
+   ```powershell
```
Comment (jluk) [inline README.md]: Change the powershell markdown to the appropriate actual scripting language (looks like bash for most)

**Review by jluk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (OGcanviz) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (OGcanviz) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (OGcanviz) [reply README.md]: updated

File: README.md (hunk 4 above)
Comment (OGcanviz) [reply README.md]: fixed

File: README.md (hunk 6 above)
Comment (OGcanviz) [reply README.md]: updated

File: README.md (hunk 7 above)
Comment (OGcanviz) [reply README.md]: updated

File: README.md (hunk 8 above)
Comment (OGcanviz) [reply README.md]: fixed

File: cmd/extracting/Dockerfile (hunk 5 above)
Comment (OGcanviz) [reply cmd/extracting/Dockerfile]: This is removed now

**Review by jluk — APPROVED** [review]

**Pull request merged**